### PR TITLE
Update dependencies.

### PR DESCRIPTION
**Pull request by XhmikosR**
* postcss to v7.0.18
* postcss-discard to v0.3.2
* read-pkg-up to v7.0.0
* xo to v0.25.3

Remove the unneeded eslint-plugin-prettier dep. xo includes this.

This has one new xo error that needs to be fixed @bezoerb.

### CODE REVIEW

**Comment by XhmikosR** [comment]
You can probably remove snyk and [enable automated security fixes](https://help.github.com/en/articles/configuring-automated-security-fixes) in all of your repos

File: src/dom.js [hunk 1]
```diff
@@ -32,9 +32,9 @@ const getPartials = (html = '', tag = 'svg') => {
   const result = [];
   html.replace(new RegExp(`<${tag}(?:\\s[^>]+)?>`, 'ig'), (match, offset, str) => {
     if (match.includes('/>')) {
-      result.push(str.substring(offset, offset + match.length));
+      result.push(str.slice(offset, offset + match.length));
```
Comment (XhmikosR) [inline src/dom.js]: Double check this change @bezoerb

Comment (bezoerb) [reply src/dom.js]: LGTM. But why do you want to switch to slice in the first place?

Comment (XhmikosR) [reply src/dom.js]: It's a new xo error https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prefer-string-slice.md

**Pull request merged**